doc/man7/flux-broker-attributes.rst

```diff
@@ -138,6 +138,15 @@ broker.exit-restart [Updates: C, R]
 broker.starttime
    Timestamp of broker startup from :man3:`flux_reactor_now`.
 
+broker.sd-notify
+   A boolean indicating that the broker should use :linux:man3:`sd_notify`
+   to inform systemd of its status.  This is set in the Flux systemd unit file.
+
+broker.sd-stop-timeout
+   The systemd TimeoutStopSec value (in RFC 23 Flux Standard Duration format),
+   which is used by the broker to extend the stop timeout while it is making
+   progress towards shutdown.  This is set in the Flux systemd unit file.
```

Comment:
Does this mean `broker.sd-stop-timeout` needs to be the same as `TimeoutStopSec`? If so, it might be clearer to just explicitly state that in this documentation. Maybe just state what the broker does with the value (which I can't remember even though I just reviewed the code :facepalm: ), then state "This value should be set equal to the `TimeoutStopSec` set for the `flux.service` unit." or similar.